admin/load.php

```diff
@@ -636,3 +639,33 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Callback function to handle admin custom script.
+ *
+ * @since n.e.x.t
+ */
+function perflab_enqueue_custom_script() {
+	?>
+<script>
+( function( $ ) {
+	if ( window.pagenow === 'settings_page_perflab-modules' ) {
```

Comment:
Can we do this check in PHP instead? No need to add JS to pages that aren't ours.